pkgs/tools/package-management/nix/default.nix

```diff
@@ -74,7 +75,18 @@ common =
           ''}
         '' else ''
           configureFlagsArray+=(BDW_GC_LIBS="-lgc -lgccpp")
```

Comment:
That could be part of `configureFlags` and outside `preConfigure`.

Reply:
This expression and the one below were already present, I added only the parens.

Reply:
Sure, but some drive-by cleanup in a follow-up commit would be more than welcome :-)

Reply:
OK. I wasn't sure if that was the case. Other OSS projects have different views on such things.

Reply:
Please make that a separate commit, so we can backport only the fixes to 20.03.